augur/filter/_run.py

```diff
@@ -489,7 +488,7 @@ def run(args):
 
     if (group_by and args.sequences_per_group) or args.subsample_max_sequences:
         seed_txt = ", using seed {}".format(args.subsample_seed) if args.subsample_seed else ""
-        print("\t%i of these were dropped because of subsampling criteria%s" % (num_excluded_subsamp, seed_txt))
+        print("\t%i were dropped because of subsampling criteria%s" % (num_excluded_subsamp, seed_txt))
```

Comment:
_non-blocking_

As a future improvement, we should totally be sending this status/summary message and all the others to stderr instead of stdout so as to keep output data separate from messages.

Reply:
There currently isn't any output data written to stdout. But yes, it would be nice to support `--output-metadata` or any other output option to stdout, which requires moving all the info messages to stderr.

Reply:
> There currently isn't any output data written to stdout

Well, you can do `--output-strains /dev/stdout` or similar and Augur will happily do it. :upside_down_face:

I often do stuff like that during testing/dev (which is why I noticed it (again) and commented), but it can be handy in actual pipelines too. Or least, it can be with good stdout/stderr discipline.

Anyway, just something for us all to keep in mind as we keep pushing Augur forward!

Reply:
Just remembered there is already an issue for this: #991